app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,92 @@
+<div class="row inset">
+<section class="col col-720">
+<p>Teaching computing is an important role to equip pupils with knowledge and skills that will help them succeed in a  world dependent on digital technology.</p>
+</section>
+<section class="col col-space-s col-content-max">
+<%= render TextBlockWithTwoImagesComponent.new(
+heading: "Help your pupils learn new skills",
+colour: "pink",
+image_1_path: "static/images/content/hero-images/0035.jpg",
+image_1_alt: "",
+image_2_path: "static/images/content/hero-images/0037.jpg",
+image_2_alt: "",
+) do %>
+<p>Computing for key stages 1 to 4 focuses on 3 pillars:</p>
```

Comment:
I'm not sure why were are mentioning key stages 1 (and 2) here - this page is about secondary not primary?